### DISCUSSION TITLE
I want to use wagmi to send a transaction, but it has an exception, how can I fix it?

### DISCUSSION THREAD

**Discussion by percyzt** [wevm/wagmi#2316] [Q&A]
![image](https://github.com/wagmi-dev/wagmi/assets/96770326/1f33a0af-1234-493b-8105-bed3d14243b2)
I want to use wagmi to send a transaction, but it has an exception, how can I fix it?

**Comment by SaadNiaziDev** [comment]
is account connected?

**Reply by percyzt** [reply]
> 帐户已连接？

I plan to use vagmi to connect the wallet, but the operation to send transactions and switch the network are both @wagmi/core, switching networks can, but sending transaction prompts cannot find the wallet client

**Reply by SaadNiaziDev** [reply]
Which version are u using and how u setup the client. If I can share code snippet it would help a lot to debug
